frontend/public/components/project-overview.jsx

```diff
@@ -0,0 +1,74 @@
+import * as React from 'react';
+import * as _ from 'lodash-es';
+import { Link } from 'react-router-dom';
+import { ListView, ListViewItem } from 'patternfly-react';
+
+import { ResourceLink } from './utils';
+import { modelFor } from '../module/k8s';
+
+export const ComponentLabel = ({text}) => <div className="co-component-label">{text}</div>;
+
+const ProjectOverviewListItem = ({obj}) => {
+  const {currentController, kind, metadata, status} = obj;
+  const {namespace, name, uid} = metadata;
+  const {latestVersion} = status;
+  const kindObj = modelFor(kind);
+  const heading = <h3 className="project-overview__item-heading">
+    <ResourceLink
+      kind={kind}
+      name={name}
+      namespace={namespace}
+    />
+  </h3>;
+
+  const additionalInfo = <div key={uid} className="project-overview__additional-info">
+    { currentController &&
+      <div className="project-overview__detail project-overview__detail--controller">
+        <ComponentLabel text={_.startCase(currentController.kind)} />
+        <ResourceLink
+          kind={currentController.kind}
+          name={currentController.metadata.name}
+          namespace={namespace}
+        />
+      </div>
+    }
+    { latestVersion &&
```

Comment:
We could consider leaving this out since it's part of the rc/rs name

Reply:
Yeah, I thought about that too. My only (very minor) concern is that we are leaving it up to the user to know that the version number is in the rc/rs name. But I don't think that's a huge deal because it's pretty obvious.